app/scripts/abos/detail/abosdetail.html

```diff
@@ -74,7 +74,7 @@ <h3 class="panel-title">
         </div>
         <div class="col-md-4">
           <p class="input-group">
-            <input type="text" class="form-control input-date" localdate datetime-picker="dd.MM.yyyy" ng-model="abo.start" is-open="open.start" enable-time="false" max-date="abo.ende" />
+            <input class="form-control input-date" localdate datetime-picker="dd.MM.yyyy" ng-model="abo.start" is-open="open.start" ng-change="updateAboDatetimePicker()" enable-time="false" datepicker-options="aboPickerStartDate.datepickerOptions" />
```

Comment:
Why did you drop type="text", the other fields still have it.

Reply:
done